src/components/ReactQueryWrapper.ts

```diff
@@ -0,0 +1,31 @@
+import * as APIFunctions from "./APIFunction"
+import { useQuery, useMutation } from "react-query"
+
+interface Comment {
+    userName: string, 
+    repoName: string, 
+    issueNumber: string, 
+    comment: string
+}
+
+export const GetUser = (userName : string) => {
+    return (useQuery(['user'], () => APIFunctions.getUser(userName)))
```

Comment:
```suggestion
    return useQuery(['user'], () => APIFunctions.getUser(userName))
```
redundant brackets

Reply:
you can also remove the curly brackets and the `return` from all these functions